charts/kong/values.yaml

```diff
@@ -282,7 +282,12 @@ proxy:
   # Enable creating a Kubernetes service for the proxy
   enabled: true
   type: LoadBalancer
-  loadBalancerClass:
+  loadBalancerClass: ""
+  # The .loadBalancerIP field for a Service was deprecated in Kubernetes v1.24.
+  # For a complete reference, visit https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer
+  loadBalancerIP: ""
```

Comment:
Given that this has been deprecated in 1.24 we probably will not want to add this at this point.

```suggestion
```